src/App.css

```diff
@@ -26,3 +26,82 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+input[type=text], input[type=tel], input[type=email], select {
+    width: 100%;
+    padding: 12px;
+    margin: 8px 0;
+    display: inline-block;
```

Comment:
display inline-block and width 100% does not make any sense to me.

Reply:
yup.. right.. My main focus was on the flow part. So, I have picked up css from w3c and just pasted it here. I'll check & correct it.